tsconfig.json

```diff
@@ -1,15 +1,14 @@
 {
     "compilerOptions": {
-      "target": "es5",
+      "target": "es2021",
```

Comment:
Without this, I had issues with TypeScript saying `string.replaceAll()` doesn't work.